tiledb/ml/models/base.py

```diff
@@ -99,7 +98,7 @@ def update_model_metadata(self, array: tiledb.Array, meta: Optional[dict] = {}):
             )
         else:
             for key, value in {**meta, **self._file_properties}.items():
-                array.meta[key] = json.dumps(value).encode("utf8")
+                array.meta[key] = value
```

Comment:
Please change also the L:220 in `tensorflow_keras.py` where we set metadata the same way. Also in case this bug fix proceeds all examples of use should also change!

Reply:
To be honest I would leave that part as is, because it's not set by the user. Moreover, this is a fix the metadata keys needed for TileDB Cloud.